ush/config_defaults.sh

```diff
@@ -1033,6 +1036,7 @@ DT_ATMOS=""
 LAYOUT_X=""
 LAYOUT_Y=""
 BLOCKSIZE=""
+FH_DFI_RADAR=(-20000000000)
```

Comment:
Is it possible to leave FH_DFI_RADAR as an empty array if expected to run without radar tten?

Reply:
We cannot set an empty value in the namelist input file. It has to fill with an value. This is default value in the namelist.

Reply:
We cannot set empty value to namelist parameters. This is default value for this namelist variable.

Reply:
suggest changing to `FH_DFI_RADAR="-20000000000"` so as to match another suggested change `FH_DFI_RADAR="0.0,0.25,0.5"`

Reply:
Does not work that way.